tests/integration/targets/meraki_ssid/tasks/main.yml

```diff
@@ -281,21 +303,23 @@
     delegate_to: localhost
     register: psk_idempotent
 
-  - debug:
+  - name: Debug psk_idempotent
+    ansible.builtin.debug:
       msg: '{{ psk_idempotent }}'
 
-  - assert:
+  - name: Assert psk_idempotent
+    ansible.builtin.assert:
       that:
         - psk_idempotent is not changed
 
   #
   # Check WPA3 Transition Mode
   #
   - name: Set WPA3 Transition Mode
```

Comment:
These tests are being skipped. I believe while Python uses the `is true` syntax, Ansible in my experience doesn't like that one as much. https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html shows they're using `not variable_name` or `variable_name` instead of `is true`.